### ISSUE TITLE
Apply PSR-2 coding style

### ISSUE THREAD

**Issue by thomaszbz** [mblaschke/TYPO3-metaseo#89]
This is a follow-up to #35.

As of 3ced6cbefbe07967bb07fb2eb0594fa96bb41905 (current develop or feature/psr2),

    /usr/bin/phpcs --standard=PSR2 --report=summary --report-width=120 .

reports

    PHP CODE SNIFFER REPORT SUMMARY
    --------------------------------------------------------------------------------
    FILE                                                            ERRORS  WARNINGS
    --------------------------------------------------------------------------------
    Classes/Backend/Ajax/AbstractAjax.php                                  1       0
    Classes/Backend/Ajax/PageAjax.php                                      8       1
    Classes/Backend/Validator/ValidatorImport.php                          2       0
    Classes/Controller/BackendPageSeoController.php                        0       1
    Classes/Controller/BackendSitemapController.php                        3       1
    Classes/Hook/Extension/NewsExtension.php                               1       0
    Classes/Hook/Extension/TtnewsExtension.php                             0       1
    Classes/Hook/SitemapIndexLinkHook.php                                  1       0
    Classes/Hook/SitemapIndexPageHook.php                                  1       0
    Classes/Page/Part/FooterPart.php                                       1       2
    Classes/Page/Part/MetatagPart.php                                      1       2
    Classes/Page/Part/PagetitlePart.php                                    1       1
    Classes/Page/RobotsTxtPage.php                                         0       1
    Classes/Scheduler/Task/AbstractSitemapTask.php                         1       0
    Classes/Scheduler/Task/SitemapTxtTask.php                              1       0
    Classes/Scheduler/Task/SitemapXmlTask.php                              2       0
    Classes/Sitemap/Generator/XmlGenerator.php                             0       2
    Classes/Utility/SitemapUtility.php                                     0       3
    Configuration/TCA/Overrides/pages.php                                  2       11
    Configuration/TCA/Overrides/pages_language_overlay.php                 3       3
    Configuration/TCA/tx_metaseo_setting_root.php                          0       15
    Resources/Public/Backend/JavaScript/MetaSeo.overview.js                3       0
    class.ext_update.php                                                   11      0
    ext_emconf.php                                                         1       2
    ext_localconf.php                                                      0       15
    --------------------------------------------------------------------------------
    A TOTAL OF 44 ERROR(S) AND 61 WARNING(S) WERE FOUND IN 25 FILE(S)
    --------------------------------------------------------------------------------

**Comment by thomaszbz** [comment]
Per 8bcdb901d628c88f62e1644a4ddad8e496f03aa5

    PHP CODE SNIFFER REPORT SUMMARY
    --------------------------------------------------------------------------------
    FILE                                                            ERRORS  WARNINGS
    --------------------------------------------------------------------------------
    Classes/Backend/Validator/ValidatorImport.php                          2       0
    Classes/Controller/BackendSitemapController.php                        3       0
    Classes/Hook/SitemapIndexPageHook.php                                  1       0
    Classes/Hook/SitemapIndexLinkHook.php                                  1       0
    Classes/Page/Part/FooterPart.php                                       1       0
    Classes/Scheduler/Task/AbstractSitemapTask.php                         1       0
    Classes/Scheduler/Task/SitemapTxtTask.php                              1       0
    Classes/Scheduler/Task/SitemapXmlTask.php                              2       0
    Resources/Public/Backend/JavaScript/MetaSeo.overview.js                3       0
    class.ext_update.php                                                   2       0
    ext_emconf.php                                                         1       1
    --------------------------------------------------------------------------------
    A TOTAL OF 18 ERROR(S) AND 1 WARNING(S) WERE FOUND IN 11 FILE(S)
    --------------------------------------------------------------------------------
